python/tests/res/fm/test_ecl_run.py

```diff
@@ -326,3 +333,19 @@ def test_error_parse(self):
         self.assertEqual( error_list[0], error0)
         self.assertEqual( error_list[1], error1)
 
+
+    @tmpdir()
+    def test_await_process_tee(self):
```

Comment:
I'm not a fan of testing private functions. Could we replace this by a test that mocks `sys.stdout`?

Reply:
I'll move the `_await_process_tee` into a new `res.util.subprocess` module as a public function.